gooddata-flight-server/setup.py

```diff
@@ -0,0 +1,69 @@
+# (C) 2021 GoodData Corporation
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text(encoding="utf-8")
+
+REQUIRES = [
```

Comment:
Can this be taken from the requirements.txt somehow? What would happen if this and requirements.txt fell out of sync?

Reply:
that's how it's done in all packages... this is the 'old style setup'.